bdd/godogs_test.go

```diff
@@ -60,8 +61,14 @@ var opts = &godog.Options{
 	StopOnFailure: true,
 	Format:        "pretty",
 	Tags:          "~@undone",
+	Strict:        true,
```

Comment:
At the moment, it might be better not to use Strict mode since we are not highly depending on bdd tests.